test/unit/validators/json-schema.test.js

```diff
@@ -71,7 +71,7 @@ describe('JsonSchema', () => {
               assert.isTrue(validator.errors instanceof ValidationErrors);
             });
 
-            it('should return some errors', () => {
+            it.only('should return some errors', () => {
```

Comment:
Oops

Reply:
There can be an eslint rule that prevents such commits. I'd suggest to adopt that to prevent such situations.

Reply:
Resolved.